### ISSUE TITLE
Improve training efficiency by integrating different sess.run()

### ISSUE THREAD

**Issue by JingyunLiang** [abhaydoke09/Bilinear-CNN-TensorFlow#21]
https://github.com/abhaydoke09/Bilinear-CNN-TensorFlow/blob/cc2df0964c8d91523f68dc5f6c96fa7bd780f06b/core/bcnn_finetuning.py#L391
Original implementation seems to propagate for  two(three) times, leading to large amount of time waste. Use 
`
sess.run([optimizer,check_op,loss,accuracy], feed_dict={imgs: batch_xs, target: batch_ys})
`
It saves nearly half of training time.
